SunEngine.Core/DataBase/DataBaseConnection.cs

```diff
@@ -17,6 +18,18 @@ namespace SunEngine.Core.DataBase
     /// </summary>
     public class DataBaseConnection : IdentityDataConnection<User, Role, int>
     {
+        public TypeSqlProvider NameSqlProvider
```

Comment:
Bad name, NameSqlProvider should be SqlProviderName  

But taking into account, that it's an enum, not a string, a better name would be `SqlProvider` or `SqlProviderType`